xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-docker/src/test/it/org/xwiki/appwithinminutes/test/ui/StaticListClassFieldIT.java

```diff
@@ -138,4 +148,12 @@ public void testItemsEditor()
         staticListField.getItemByValue("value1").click();
         assertEquals(Arrays.asList("XWiki", "value1"), staticListField.getDefaultSelectedValues());
     }
+
+    private ApplicationClassEditPage goToEditor(TestUtils testUtils, EntityReference entityReference)
```

Comment:
Could be added to ApplicationClassEditPage PO no?

Reply:
indeed, good idea. See https://github.com/xwiki/xwiki-platform/pull/1570/files#diff-d81595374301920b258e062817f140984848a0be52c7cc8db74c38ba6ccbcd69R147-R152